src/components/cards/DataSummaryCardGroup.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import CountBox from './CountBox';
+import './DataSummaryCardGroup.less';
+
+/**
+ * Little card with a bunch of counters on it for cases, experiments, files, ...
+ */
+class DataSummaryCardGroup extends Component {
+  render() {
+    const totalWidth = typeof this.props.width === 'string' || `${this.props.width}px`;
+    return (
+      <div
+        className="data-summary-card-group"
+        style={{
+          width: totalWidth,
+          height: this.props.height,
+        }}
+      >
+        {
+          this.props.summaryItems.map((item, index) => (
+            <div
+              className={'data-summary-card-group__card'.concat(this.props.connected ? ' connected' : ' separated')}
+              key={item.label || item[0].label}
+            >
+              {this.props.connected && index > 0
+                && <div className="data-summary-card-group__card-left-border" />
+              }
+              {
+                !item.length ? (
+                  <CountBox
+                    label={item.label}
+                    value={item.value}
+                    align={this.props.align}
+                  />
+                ) : (
+                  <div className="data-summary-card-group__sub-card-group">
+                    {
+                      item.map((subItem, subIndex) => (
+                        <div className="data-summary-card-group__sub-card-item" key={subItem.label}>
+                          {subIndex > 0
+                            && <div className="data-summary-card-group__sub-card-left-border" />
```

Comment:
Why does there need to be a separate class for this?

Reply:
The style is different for sub-card and card so I created 2 classes